src/support/Dhis2.js

```diff
@@ -31,321 +31,321 @@ class Dhis2 {
     if (orgunits && orgunits.length === 1) {
-      searchOuUrl += "&filter=path:like:" + orgunits[0].id;
+      searchOuUrl += '&filter=path:like:' + orgunits[0].id
     } else if (orgunits && orgunits.length > 0) {
       searchOuUrl +=
-        "&filter=ancestors.id:in:[" + orgunits.map(ou => ou.id).join(",") + "]";
+        '&filter=ancestors.id:in:[' + orgunits.map(ou => ou.id).join(',') + ']'
     }
-    return getInstance().then(d2 => d2.Api.getApi().get(searchOuUrl));
+    return getInstance().then(d2 => d2.Api.getApi().get(searchOuUrl))
   }
 
   getAllDataElements(userIds) {
     var dataElementsUrl =
-      "/dataElements.json?fields=id,name,valueType,domainType,user[id,name]&paging=false&filter=user.id:in:[" +
-      userIds.join(",") +
-      "]";
+      '/dataElements.json?fields=id,name,valueType,domainType,user[id,name]&paging=false&filter=user.id:in:[' +
+      userIds.join(',') +
+      ']'
 
-    return getInstance().then(d2 => d2.Api.getApi().get(dataElementsUrl));
+    return getInstance().then(d2 => d2.Api.getApi().get(dataElementsUrl))
   }
 
   getDataElementNames(dataElementGroup) {
     var dataElementsUrl =
-      "dataElementGroups/" +
+      'dataElementGroups/' +
       dataElementGroup +
-      ".json?fields=dataElements[id,name]";
+      '.json?fields=dataElements[id,name]'
 
-    return getInstance().then(d2 => d2.Api.getApi().get(dataElementsUrl));
+    return getInstance().then(d2 => d2.Api.getApi().get(dataElementsUrl))
   }
 
   getDataElementNamesByGroups(dataElementGroupIds) {
     var dataElementsUrl =
-      "dataElements.json?pageSize=1000&filter=dataElementGroups.id:in:[" +
-      dataElementGroupIds.join(",") +
-      "]";
+      'dataElements.json?pageSize=1000&filter=dataElementGroups.id:in:[' +
+      dataElementGroupIds.join(',') +
+      ']'
 
-    return getInstance().then(d2 => d2.Api.getApi().get(dataElementsUrl));
+    return getInstance().then(d2 => d2.Api.getApi().get(dataElementsUrl))
   }
 
   getDataElementNamesByDataSets(dataSetIds) {
     var dataElementsUrl =
-      "dataElements.json?pageSize=1000&filter=dataSetElements.dataSet.id:in:[" +
-      dataSetIds.join(",") +
-      "]";
+      'dataElements.json?pageSize=1000&filter=dataSetElements.dataSet.id:in:[' +
+      dataSetIds.join(',') +
+      ']'
 
-    return getInstance().then(d2 => d2.Api.getApi().get(dataElementsUrl));
+    return getInstance().then(d2 => d2.Api.getApi().get(dataElementsUrl))
   }
 
   buildInvoiceRequest(orgUnits, period, invoiceType, orgUnitId) {
-    const year = period.slice(0, 4);
-    const quarter = DatePeriods.split(period, "quarterly")[0].slice(5, 6);
+    const year = period.slice(0, 4)
+    const quarter = DatePeriods.split(period, 'quarterly')[0].slice(5, 6)
 
     return {
       orgUnit: orgUnits.filter(orgUnit => orgUnit.id === orgUnitId)[0],
       orgUnits: orgUnits,
       period: period,
       quarterPeriod: period,
       quarterPeriods: DatePeriods.split(period, invoiceType.frequency),
```

Comment:
quarterPeriods should be dhis2 periods quarters

Reply:
@mestachs I have fixed the "prettier" configuration in atom
![selection_002](https://user-images.githubusercontent.com/19631540/51370048-616c1780-1afe-11e9-885f-13de84c47a78.png)

Reply:
IMO src/support/Dhis2.js
line 302 : quarterPeriods: DatePeriods.split(period, "quarterly"),

Reply:
Can you prettyfy this PR with the new settings ?